lib/perl/Genome/Model/ClinSeq/Command/MicroarrayCnv.pm

```diff
@@ -321,14 +412,21 @@ sub run_cnview {
         @cnv_symbols = qw (Kinase_dGene CancerGeneCensusPlus_Sanger AntineoplasticTargets_DrugBank All);
     }
     my $gene_symbol_dir = $cancer_annotation_db->data_directory . "/GeneSymbolLists/";
+
+    #Run only chr6 for test-cases
+    my $plot_chr = "ALL";
+    if($self->test) {
+        $plot_chr = "6";
+    }
+
     foreach my $symbol(@cnv_symbols){
         my $symbol_outdir =  $self->outdir;
         if ($symbol eq "All") {
-            my $cnview_cmd = Genome::Model::Tools::CopyNumber::CnView->create(annotation_build => $self->annotation_build_id, cnv_file => $cnv_hq_file, segments_file => $cnv_hmm_file, output_dir => $symbol_outdir, name => $symbol, cancer_annotation_db => $cancer_annotation_db, window_size => 0, verbose => 1);
+            my $cnview_cmd = Genome::Model::Tools::CopyNumber::CnView->create(annotation_build => $self->annotation_build_id, cnv_file => $cnv_hq_file, segments_file => $cnv_hmm_file, output_dir => $symbol_outdir, name => $symbol, cancer_annotation_db => $cancer_annotation_db, window_size => 0, verbose => 1, chr => $plot_chr);
             $cnview_cmd->execute();
         } else {
             my $gene_targets_file = "$gene_symbol_dir/$symbol" . ".txt";
-            my $cnview_cmd = Genome::Model::Tools::CopyNumber::CnView->create(annotation_build =>  $self->annotation_build_id, cnv_file => $cnv_hq_file, segments_file => $cnv_hmm_file, output_dir => $symbol_outdir, gene_targets_file => $gene_targets_file, name => $symbol, cancer_annotation_db => $cancer_annotation_db, window_size => 0, verbose => 1);
+            my $cnview_cmd = Genome::Model::Tools::CopyNumber::CnView->create(annotation_build =>  $self->annotation_build_id, cnv_file => $cnv_hq_file, segments_file => $cnv_hmm_file, output_dir => $symbol_outdir, gene_targets_file => $gene_targets_file, name => $symbol, cancer_annotation_db => $cancer_annotation_db, window_size => 0, verbose => 1, chr => $plot_chr);
```

Comment:
If these two differ only in `gene_targets_file`, the duplication of these params listings could be removed using something like:

```perl
my %params = (annotation_build => ..., chr => $plot_chr);
if ($symbol ne 'All') {
  $params{gene_targets_file} = "$gene_symbold_dir/$symbol" . ".txt";
}
my $cnview_cmd = Genome::Model::Tools::CopyNumber::CnView->create(%params);
$cnview_cmd->execute;
```